File: eDental-flutter-main/lib/models/dentist.dart
import 'dart:typed_data';
import 'package:dart_json_mapper/dart_json_mapper.dart';

@jsonSerializable
class Dentist {
  int id;
  String firstName;
  String lastName;
  String phone;
  String email;
  String address;
  DateTime birthDate;
  int dentistOfficeId;
  Uint8List image;
  String description;
  bool active;
  String fullName;

  @override
  bool operator ==(other) => other is Dentist && id == other.id;

  Dentist(
    this.id,
    this.firstName,
    this.lastName,
    this.phone,
    this.email,
    this.address,
    this.birthDate,
    this.dentistOfficeId,
    this.image,
    this.description,
    this.active,
    this.fullName,
  );
}

File: edentalbackend-main/DentalOffice.Entities/dbEntities/Dentist.cs
namespace DentalOffice.Entities.dbEntities
{
    public class Dentist
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public DateTime BirthDate { get; set; }
        public byte[] Image { get; set; }
        public string Description { get; set; }
        public bool Active { get; set; }
        public int DentistOfficeId { get; set; }
        public DentistOffice DentistOffice { get; set; }
    }
}
